integrations/oci/sdk/runtime/src/test/resources/test-oci-config-file.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# should resolve to "config-file" as the configured (by probably not available) auth strategy
+auth-strategies: "config-file, resource-principal, config, instance-principals"
```

Comment:
Comma separated valules, with spaces is not something I expect to see in our configuration file.
This looks like an array - it should be defined as array, e.g.:
```yaml
auth-strategies: ["config-file", "resource-principal", "config", "instance-principals"]
```

Also is it correct to have `resource-principal` as singular, and `instance-principals` as plural? This used to be both plural in original code, not sure why changed.

Reply:
Also why do we have support for `auth-strategies` and `auth-strategy`? It should be enough to have one option that has one or more values. Or what is the meaning here?

Reply:
the plural vs singular matches the oci style.  i know it is strange.  for the strategies and strategy I matched the legacy implementation that Laird did.

Reply:
> Also why do we have support for auth-strategies and auth-strategy?

I also don't know why that is in here. In the CDI `OciExtension` that existed because an old version of `OciExtension` "got into the wild" with `auth-strategy`, so it was a fallback value to preserve backwards compatibility. `auth-strategies` is the preferred, documented way to indicate a list of authentication strategies; if this mechanism here wants to duplicate it it shouldn't have to worry about the old `auth-strategy` fallback.